### PR TITLE
Add edit and list sequence diagram

### PR DESCRIPTION
**Pull request by wish2023**

### CODE REVIEW

File: docs/sequenceDiagrams/ListSequenceDiagramCode.txt [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+activate TaskList
+deactivate TaskList
+
+TaskList-->-Timetable:
+
+Timetable->Timetable: populateTimetable()
+activate Timetable
+deactivate Timetable
+
+Timetable-->-ListCommand:
```
Comment (Kafcis) [inline docs/sequenceDiagrams/ListSequenceDiagramCode.txt:27]: Basic functions are covered that's good. Cleaned up the diagram a little bit, check it out and tell me what you think


title List

actor ATHENA

ATHENA->ListCommand: execute()
activate ListCommand

ListCommand->*ImportanceFilter:
activate ImportanceFilter
ImportanceFilter-->ListCommand:
destroy ImportanceFilter

ListCommand->*ForecastFilter: 
activate ForecastFilter
ForecastFilter-->ListCommand:
deactivate ForecastFilter


ListCommand->*Timetable:
activate Timetable

Timetable->+TaskList: getFilteredList()
TaskList->+TaskFilter: isTaskIncluded()
TaskFilter-->-TaskList:
TaskList->+ForecastFilter: removeExcludedDates()
ForecastFilter-->-TaskList:

destroy ForecastFilter
destroy TaskFilter



TaskList-->-Timetable:

Timetable->Timetable: populateTimetable()
activate Timetable
deactivate Timetable

Timetable-->-ListCommand:
ListCommand-->-ATHENA:

File: docs/sequenceDiagrams/EditSequenceDiagramCode.txt [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+EditCommand->+TaskList: editTask()
+
+TaskList->TaskList: getTaskFromNumber()
+activate TaskList
+deactivate TaskList
+
+TaskList->TaskList: createTask()
+activate TaskList
+TaskList->*+Task: <<create>>
```
Comment (Kafcis) [inline docs/sequenceDiagrams/EditSequenceDiagramCode.txt:13]: Rest of this looks good, one small suggestion to change this to TaskList->*Task:  , it makes the arrow point directly to the block at the top.

File: docs/sequenceDiagrams/EditSequenceDiagramCode.txt [hunk 3]
```diff
@@ -0,0 +1,26 @@
+title Edit
+
+
```
Comment (Kafcis) [inline docs/sequenceDiagrams/EditSequenceDiagramCode.txt:3]: Might want to include the actor here

**Review by Kafcis — CHANGES_REQUESTED** [review]

File: docs/sequenceDiagrams/EditSequenceDiagramCode.txt (hunk 2 above)
Comment (wish2023) [reply docs/sequenceDiagrams/EditSequenceDiagramCode.txt:13]: This gives me a syntax error for some reason

File: docs/sequenceDiagrams/EditSequenceDiagramCode.txt (hunk 3 above)
Comment (wish2023) [reply docs/sequenceDiagrams/EditSequenceDiagramCode.txt:3]: Oh hmmm I thought it may be better to include the actor for the parser?

**Pull request closed without merging**